### PR TITLE
must-gather: collect timestamp and total time taken by the MG

### PR DESCRIPTION
**Pull request by rajatsing**
this pr will add the feature to show the total time take by the collection script.

Signed-off-by: RAJAT SINGH <rajasing@redhat.com>

### CODE REVIEW

**Review by jarrpa — APPROVED** [review]
/lgtm

**Review by crombus — CHANGES_REQUESTED** [review]
Code changes looks good.
Just rebase and add the time diff to gather-debug log

File: must-gather/collection-scripts/gather [hunk 1]
```diff
@@ -24,6 +29,16 @@ fi
...
 
-echo "deleting empty files" >> ${BASE_COLLECTION_PATH}/gather-debug.log
+# timestamp for ending of the script
+END_TIME="$(date +%r)"
+date2=$(date +"%s")
+diff=$("${date2}"-"${date1}")
+printf "collection ended at: %s \n" "${END_TIME}" >> ${BASE_COLLECTION_PATH}/gather-debug.log 2>&1
+
+# collecting the total time taken by the scrript script
+total_time=$(("${diff}" / 3600 )) hours $(("${diff}" % 3600) / 60)) minutes $(("${diff}" % 60)) seconds
```
Comment (rajatsing) [inline must-gather/collection-scripts/gather:39]: @crombus my CI is failing here, need some help in what pattern am I missing.

Comment (rajatsing) [reply must-gather/collection-scripts/gather:39]: Getting this alert
```
 ^-- SC1073: Couldn't parse this $((..)) expression. Fix to allow more checks.
 ^-- SC1009: The mentioned syntax error was in this $((..)) expression.
^-- SC1072: Expected a double )) to end the $((..)). Fix any mentioned problems and try again.                                                       
```

Comment (rajatsing) [reply must-gather/collection-scripts/gather:39]: This line is also not getting printed in the gather debug log file, probably due to this error that I'm getting?.

Comment (crombus) [reply must-gather/collection-scripts/gather:39]: total_time=$(("${diff}" / 3600 )) hours $((("${diff}" % 3600) / 60)) minutes $(("${diff}" % 60)) seconds

Comment (crombus) [reply must-gather/collection-scripts/gather:39]: update the brackets in minutes column.

**Review by crombus — CHANGES_REQUESTED** [review]

**Review by jarrpa — APPROVED** [review]
/lgtm

**Pull request merged**